### DISCUSSION TITLE
Unable to apply for Coplit

### DISCUSSION THREAD

**Discussion by UsmanKhalilKIET** [community/community#121523] [GitHub Education]
### Select Topic Area

Question

### Body

Dear Sir/Madam,

Hope that you are doing well.

I am applying for Github Copilot for teachers but I am receiving an error message that the uploaded proof has already been used previously. (The picture is given below for your assistance).Kindly let me know how I can resolve this issue and apply for Github Coplit.
Thank you.

best regards,
Usman Khalil
![Untitled](https://github.com/community/community/assets/168419418/c5794175-0939-4c9a-90ac-cac21dc746b5)

**Comment by itzzjb** [comment]
You can take a more clear picture of the same document. Crop the image until the borders of the document. Then try to upload it.  But don't try to upload the same image. Also mind that you need to upload a .jpeg or a .png file.

**Comment by UsmanKhalilKIET** [comment]
Ok, I will do that.
Thanks.
